Content.Server.Database/Model.cs

```diff
@@ -1119,4 +1120,16 @@ public class RoleWhitelist
         [Required]
         public string RoleId { get; set; } = default!;
     }
+
+    [Table("text_automod")]
```

Comment:
You don't need to specify this name explicitly anymore.

Reply:
Done [b07985b](https://github.com/space-wizards/space-station-14/pull/28570/commits/b07985be4990b4db2ac41a65ae20c87c1eff669a)